autoPyTorch/utils/common.py

```diff
@@ -35,6 +46,25 @@ def __str__(self) -> str:
             self.name, self.supported_types, self.user_defined, self.dataset_property)
 
 
+class HyperparameterSearchSpace(NamedTuple):
+    """
+    A class that holds the search space for an individual hyperparameter.
+    Attributes:
+        hyperparameter (str):
+            name of the hyperparameter
+        value_range (Sequence[HyperparameterValueType]):
+            range of the hyperparameter
```

Comment:
Maybe here clarify that it can be a rangle between max-min or a range of possible choices